### PR TITLE
Refactoring and minor change.

### PR DESCRIPTION
**Pull request by ghost**
- Modify not working disconnect function.
- Rename dynamic value for fitting convention.
- Refactoring check-timer function.

### CODE REVIEW

File: src/mqttkat/client.clj [hunk 1]
```diff
@@ -80,9 +80,9 @@
 
 (defn disconnect [client]
   (let [map (gen/generate (s/gen :mqtt/disconnect))
-        bufs (MqttDisconnect/encode map)]
+        bufs (MqttDisconnect/encode)]
```
Comment (thomas-shares) [inline src/mqttkat/client.clj]: Can't we get rid of the map completely in this case? it isn't used later on anyway. And then we can get rid of the `let` as well.

**Review by thomas-shares — CHANGES_REQUESTED** [review]
WOW!!!!!!
Thank you very much indeed for doing this. I haven't looked at this project for a while I have to admit. And I love the changes you made and I think I have spotted another small improvement.

File: src/mqttkat/client.clj (hunk 1 above)
Comment (ghost) [reply src/mqttkat/client.clj]: Ok, I will remove unused feature.

Comment (ghost) [reply src/mqttkat/client.clj]: I removed it.

**Comment by thomas-shares** [comment]
Thank you for the first ever Pull Request on this Repo!!!! I really appreciate this very much!!!!

**Pull request merged**